testdata/p4_16_samples/actions-almost-duplicate-names1.p4

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
@chrispsommers @smolkaj This is the second of the two new P4 test programs I would recommend you focus your review on.  The comments indicate which parts of the program I expect should _not_ give errors, and they do not.  You might also want to look through the contents of the expected generated P4Info file named testdata/p4_16_samples_outputs/actions-almost-duplicate-names1.p4.p4info.txtpb here: https://github.com/p4lang/p4c/pull/4975/files#diff-958b36defef5fa0260fb1c4424a06e6f725d05c1853b99070b691a48ee458c7b